api/admin/adminModel.js

```diff
@@ -4,20 +4,20 @@ const getAdmins = async () => {
   return await db('admins').leftJoin(
     'profiles',
     'admins.user_id',
-    'profiles.key'
+    'profiles.okta'
   );
 };
 
 const findByAdminId = async (id) => {
   return db('admins')
-    .leftJoin('profiles', 'admins.user_id', 'profiles.key')
```

Comment:
Looks like we just changed everything from 'hey' to 'okta' to match the tables right?

Reply:
Yep!